test/e2e-test.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+function install_pipeline_crd() {
+  local latestreleaseyaml
+  echo ">> Deploying Tekton Pipelines"
+  if [[ -n ${RELEASE_YAML} ]];then
+  latestreleaseyaml=${RELEASE_YAML}
+  else
+  latestreleaseyaml=$(curl -s https://api.github.com/repos/tektoncd/pipeline/releases|python -c "import sys, json;x=json.load(sys.stdin);ass=x[0]['assets'];print([ x['browser_download_url'] for x in ass if x['name'] == 'release.yaml'][0])")
+  fi
+  [[ -z ${latestreleaseyaml} ]] && fail_test "Could not get latest released release.yaml"
+  kubectl apply -f ${latestreleaseyaml} ||
+  fail_test "Tekton pipeline installation failed"
+
+  # Make sure that eveything is cleaned up in the current namespace.
+  for res in pipelineresources tasks pipelines taskruns pipelineruns; do
+  kubectl delete --ignore-not-found=true ${res}.tekton.dev --all
+  done
+
+  # Wait for pods to be running in the namespaces we are deploying to
+  wait_until_pods_running tekton-pipelines || fail_test "Tekton Pipeline did not come up"
+}
+
+function install_manual_approval_crd() {
+  echo ">> Deploying Manual Approval Gate"
+  ko apply -f config/kubernetes
```

Comment:
you need the same `ko` flags here I think.

Reply:
So I have not called this function in the main, meaning i have commented that, directly used in the actions to check if the pods are getting up